src/main/java/org/gitlab4j/api/webhook/EventChanges.java

```diff
@@ -22,6 +23,7 @@ public abstract class EventChanges {
     private ChangeContainer<Long> milestoneId;
     private ChangeContainer<List<EventLabel>> labels;
     private ChangeContainer<List<Assignee>> assignees;
+    private ChangeContainer<List<Reviewer>> reviewers;
```

Comment:
I am not sure this is correct here.

This class is shared between `IssueChanges` and `MergeRequestChanges` and the `reviewers` attribute is something that only makes sense in the context of a MR.

So I think it should be added only to `MergeRequestChanges`.